<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1087</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="devicesNamesList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>631</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="vendorIDList">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>20</y>
      <width>211</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="deviceIDList">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>20</y>
      <width>191</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
